tests/queries/0_stateless/01523_client_local_queries_file_parameter.sh

```diff
@@ -21,4 +21,9 @@ DROP TABLE 01523_test;" > 01523_client_local_queries_file_parameter_tmp.sql
 
 echo -e "1,2\n3,4" | $CLICKHOUSE_LOCAL --queries-file=01523_client_local_queries_file_parameter_tmp.sql 2>&1
 
+# Simultaneous --queries-file + --query is prohibited.
```

Comment:
Cosmetic: let's add `passing` between `Simultaneously` and `--queries-file`.

Reply:
Done.